### ISSUE TITLE
[Bug] Notification loop

### ISSUE THREAD

**Issue by HistoryCucumber** [YTVanced/Vanced#1181]
**Bug description**
I keep getting the same notifications over and over again. (Also on a side note there should be a voip role in the discord for people with voip numbers)

**Variant**
Non-Root

**Vanced version**
The one that added sponsorblock

**Android version**
Android 9

**Device**
Moto X4

**Steps to Reproduce**
Isn't that hard just be subscribed to some people

**Expected behavior**
Getting the notifications only once

**Comment by SypherRed** [comment]
Same does happen to me with YT Vanced Version: 16.29.39 using Samsung S6 Edge with Android 7.0

**Comment by milindgoel15** [comment]
duplicate of #1107
